lib/chat_message_updater.rb

```diff
@@ -21,21 +21,29 @@ def initialize(chat_message:, new_content:, upload_ids: nil)
   def update
     begin
       @chat_message.message = @new_content
-        @chat_message.cook
-        @chat_message.save!
-        revision = save_revision!
-        update_uploads!
-        ChatPublisher.publish_edit!(@chat_channel, @chat_message)
-        Jobs.enqueue(:process_chat_message, { chat_message_id: @chat_message.id })
-        DiscourseChat::ChatNotifier.notify_edit(chat_message: @chat_message, timestamp: revision.created_at)
-      rescue => error
-        @error = error
-        if Rails.env.test?
-          puts "#" * 50
-          puts "Chat message update error:"
-          puts @error.inspect
-          puts "#" * 50
-        end
+      validate_message!
+      @chat_message.cook
+      @chat_message.save!
+      revision = save_revision!
+      update_uploads!
+      ChatPublisher.publish_edit!(@chat_channel, @chat_message)
+      Jobs.enqueue(:process_chat_message, { chat_message_id: @chat_message.id })
+      DiscourseChat::ChatNotifier.notify_edit(chat_message: @chat_message, timestamp: revision.created_at)
+    rescue => error
+      @error = error
+      if Rails.env.test?
+        puts "#" * 50
+        puts "Chat message update error:"
+        puts @error.inspect
+        puts "#" * 50
+      end
```

Comment:
Not sure this needs to be in the commit?

Reply:
That is just a whitespace change.. it's been in there. I find it helpful for local debugging as errors are swallowed in the `rescue` if you have a syntax error or something like that.